httpserver/handlers.go

```diff
@@ -118,7 +120,7 @@ func (server *Server) verify(ctx context.Context, w http.ResponseWriter, r *http
 				}
 
 				if res, err := json.MarshalIndent(result, "", "  "); err == nil {
-					fmt.Println(string(res))
+					logrus.Infof("verify result for subject %v: %+v", resolvedSubjectReference, res)
```

Comment:
Does this output in the logs as a multi line format like `fmt.Println` does?

Reply:
Yeah, I tried both ways which have the same behavior.